pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "metis-client"
-dynamic = ["version"]  # read from metis_client/__init__.py
+version = "0.0.3"
```

Comment:
@knopki no more read from `metis_client/__init__.py` ?

Reply:
`commitizen` should update this on bump, so dynamic import is useless.